content/tracing/_index.md

```diff
@@ -40,6 +40,7 @@ For additional information, reference [the Docker page](/tracing/docker)
 To instrument your application, select one of the following supported languages.
 
 - [Go](/tracing/go)
```

Comment:
you should update all languages link. `/tracing/go` => `/tracing/languages/go` :)

Reply:
I added an alias to the java page so now they're all consistent. Is this okay instead?

Reply:
we should avoid redirects on the documentation. Having and alias is a good thing (just in case) but we should update the language link.

Reply:
Sounds good, I'll make the changes. 👍